src/_data/site.yml

```diff
@@ -1,5 +1,6 @@
 images:
   roundel: https://raw.githubusercontent.com/cs-soc-tudublin/IdentityFlip/master/assets/logos/cspp.svg
+  png: https://raw.githubusercontent.com/cs-soc-tudublin/IdentityFlip/master/assets/logos/roundel.png
```

Comment:
`png` seems a little non-descriptive as a variable name. Especially given there's another png in the same category. Maybe `roundel_png` and the other can be `roundel_svg`?

Reply:
Would calling it embedImage work?